src/components/Comments/Comments.jsx

```diff
@@ -34,24 +34,34 @@ const Comments = () => {
   ]);
 
   useEffect(() => {
-    setComments(
-      comments.map((comment) => (
-        <Comment
-          author={comment.author}
-          text={comment.text}
-          date={comment.date}
-          image={comment.image}
-          key={comment.id}
-        />
-      )),
-    );
+    // setComments(
+    //   comments.map((comment) => (
+    //     <Comment
```

Comment:
I am using my own Comment component. This code is "fictional". Idea is, to get comments on client-side in the future, that's why I put this code in useEffect function